search/search_service/api/search.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from http import HTTPStatus
+from typing import (  # noqa: F401
+    Any, Iterable, List,
+)
+
+from amundsen_common.models.search import SearchRequestSchema
+from flasgger import swag_from
+from flask_restful import Resource, request
+
+from search_service.proxy.es_search_proxy import (
+    RESOURCE_STR_MAPPING, ElasticsearchProxy, Resource as AMDResource,
```

Comment:
AMD is not an obvious abbreviation for me, we could go just with `AmundsenResource`